src/app/static/src/app/store/baseline/mutations.ts

```diff
@@ -38,6 +38,11 @@ export const BaselineUpdates = [
     BaselineMutation.PopulationUpdated
 ];
 
+export const BaselineDatasetUpdates = [
+    BaselineMutation.SetDataset,
+    BaselineMutation.SetRelease
```

Comment:
So options will be cleared when either of these are updated? Don't we want to retain them on `SetRelease`? That's equivalent to loading a 'different version of the dataset' . Or are we only expecting to retain options when we do a refresh when the latest version changes?